packages/system/src/themes/solidjs-color-scheme.ts

```diff
@@ -0,0 +1,84 @@
+import { ColorPalette } from "./theme-from-palette"
+
+export const solidJSColorPalette: ColorPalette = {
+	highlight: "#2c4f7c",
+	logo: "#457ABF",
+	primary100: "#ffffff",
```

Comment:
Just for clarity. How was this colour palette derived? Were you provided with this or its random?

Reply:
extracted the color palette from the solidJS website, using this tool https://www.colorcombos.com

But i think we'd probably need to get these colors using the same method that was used to generated the others.

Reply:
I agree. 
You can ask Dustin